addons/product/product_view.xml

```diff
@@ -801,29 +829,34 @@
             <field name="model">product.supplierinfo</field>
             <field name="arch" type="xml">
                 <form string="Vendor Information">
-                    <group col="4">
-                        <field name="name" context="{'default_customer': 0, 'search_default_supplier': 1, 'default_supplier': 1}"/>
-                        <field name="sequence"/>
-                        <field name="product_name"/>
-                        <field name="product_code"/>
-                        <field name="min_qty"/>
-                        <field name="product_uom" groups="product.group_uom"/>
-                        <field name="delay"/>
-                        <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                    <group>
+                        <group string="Vendor">
+                            <field name="name" context="{'default_customer': 0, 'search_default_supplier': 1, 'default_supplier': 1}"/>
+                            <field name="product_name"/>
+                            <field name="product_code"/>
+                            <label for="delay"/>
+                            <div>
+                                <field name="delay" class="oe_inline"/> days
+                            </div>
+                        </group>
+                        <group string="Price List">
+                            <field name="product_tmpl_id" string="Product" invisible="context.get('visibile_product_tmpl_id', True)"/>
+                            <label for="min_qty"/>
+                            <div>
+                                <field name="min_qty" class="oe_inline"/>
+                                <field name="product_uom" class="oe_inline" groups="product.group_uom"/>
+                            </div>
+                            <label for="price"/>
+                            <div>
+                                <field name="price" class="oe_inline"/><field name="currency_id" class="oe_inline" groups="base.group_multi_currency"/>
+                            </div>
+                            <label for="date_start" string="Validity"/>
+                            <div><field name="date_start" class="oe_inline"/> to <field name="date_end" class="oe_inline"/></div>
+                        </group>
+                        <group string="Other Information">
+                            <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
```

Comment:
what happen if multi company group is not enable.? give groups in group tag. like <group groups="base.group_multi_company">